client/components/common/tce-core/ElementList.vue

```diff
@@ -42,21 +41,13 @@ export default {
     layout: { type: Boolean, default: false },
     enableAdd: { type: Boolean, default: true }
   },
-  data() {
-    return { dragElementIndex: null };
-  },
+  data: () => ({ dragElementIndex: null, showElementDrawer: false }),
   computed: {
-    options() {
-      return {
-        handle: '.drag-handle',
-        scrollSpeed: 15,
-        scrollSensitivity: 125
-      };
-    },
-    nextPosition() {
-      const lastItem = last(this.elements);
-      return lastItem ? lastItem.position + 1 : 1;
-    }
+    options: () => ({
```

Comment:
Do need this computed here? Maybe just move the object in the template `:option` prop.